lib/data-privacy.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var
+    Private = require('./private.js'),
+    {
+        KlaviyoError
+    } = require('./errors.js');
+
+class DataPrivacy extends Private {
+    /**
+     * Request a data privacy-compliant deletion for the person record corresponding to an email address, phone number, or person identifier.
+     * https://www.klaviyo.com/docs/api/v2/data-privacy#post-deletion-request
+     * @param email
+     * @param phone_number
+     * @param person_id
+     * @returns {Promise<Object|KlaviyoApiError>}
+     */
+    requestProfileDeletion({ email, phone_number, person_id } = {}) {
```

Comment:
arguments shoud just be the id type and the id value, then make sure the id type is an allowed type